homeassistant/components/freebox/home_base.py

```diff
@@ -77,19 +79,38 @@ async def async_update_signal(self):
             )
         self.async_write_ha_state()
 
-    async def set_home_endpoint_value(self, command_id: Any, value=None) -> None:
+    async def set_home_endpoint_value(self, command_id: Any, value=None) -> bool:
         """Set Home endpoint value."""
         if command_id is None:
             _LOGGER.error("Unable to SET a value through the API. Command is None")
-            return
-        await self._router.home.set_home_endpoint_value(
-            self._id, command_id, {"value": value}
-        )
+            return False
+        try:
+            await self._router.home.set_home_endpoint_value(
+                self._id, command_id, {"value": value}
+            )
+        except HttpRequestError as error:
```

Comment:
Why this exception is raised?
Id there a normal use case where the exception should be handled ?

Reply:
Sometimes I have an exception... perhaps network or Freebox issues